### PR TITLE
Greediness

### PR DESCRIPTION
**Pull request by Airblader**

### CODE REVIEW

**Comment by Airblader** [comment]
@psychon So, this would fix the last 1.0-relevant issue (AFAICT). I'll say this isn't the *prettiest* code in the world, though. Thoughts?

File: include/match.h [hunk 1]
```diff
@@ -35,7 +35,7 @@
 
 /** Information about a matched component. */
 typedef enum xcb_xrm_match_flags_t {
-    MT_NONE = 1 << 0,
+    MF_NONE = 1 << 0,
```
Comment (psychon) [inline include/match.h]: What a useless bit... This bit is set to indicate... nothing? ;-)

**Comment by psychon** [comment]
If it works, who am I to complain? :-)
I agree that I'd be nice to have this in a nicer way, but at least it works. Thanks!

File: include/match.h (hunk 1 above)
Comment (Airblader) [inline include/match.h]: Yes! :) Although I'm just now thinking that maybe there should be a function that does the match on a single component and returns the match flags. Then we can avoid the duplication of "did it match?" we have now – and then we'd also actually need this bit.

**Comment by Airblader** [comment]
Alright. =) I've pushed another commit which cleans this up rather nicely now, I think.

**Pull request merged**